Game.java

```diff
@@ -7,10 +7,11 @@
 
     private InputStream stream;
     private Board board;
-    private String gameMode = "pvp";
+    public String gameMode = "pvp";
     private Player playerActive;
     private Player playerDeactive;
     private BufferedReader inputReader;
+    private ConsoleGame consoleGame = new ConsoleGame(this);
 
```

Comment:
By doing this you are composing Game with a ConsoleGame, dependencies should be injected into the constructor.